LoopKitUI/Views/TherapySettingsView.swift

```diff
@@ -0,0 +1,342 @@
+//
+//  TherapySettingsView.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 7/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import AVFoundation
+import HealthKit
+import LoopKit
+import SwiftUI
+
+public protocol TherapySettingsViewDelegate: class {
+    func gotoEdit(therapySetting: TherapySetting)
```

Comment:
So TherapySettingsViewDelegate is managing navigation externally, by pushing onto a UINavigationController?
Is this necessary, or does SwiftUI's navigation API suffice?

Reply:
Two reasons:

1) It isn't necessarily the case that we will be using a NavigationLink because we're still ironing out the design.
2) With this view "living" in two worlds (onboarding & settings), I wanted to delegate the navigation responsibility to an outer class to deal with navigation, possibly in different ways for the two different situations.

Reply:
(Also, this is all still work-in-progress)